File: src/main/java/network/matic/dagger/Options.java
package network.matic.dagger;

import org.eclipse.paho.client.mqttv3.MqttClientPersistence;
import org.eclipse.paho.client.mqttv3.MqttConnectOptions;

public class Options {
    private String clientId;
    private Callback callback;
    private MqttConnectOptions mqttConnectOptions;
    private MqttClientPersistence mqttClientPersistence;

    public Options() {}

    public MqttConnectOptions getMqttConnectOptions() {
        return mqttConnectOptions;
    }

    public void setMqttConnectOptions(MqttConnectOptions mqttConnectOptions) {
        this.mqttConnectOptions = mqttConnectOptions;
    }

    public Callback getCallback() {
        return callback;
    }

    public void setCallback(Callback callback) {
        this.callback = callback;
    }

    public String getClientId() {
        return clientId;
    }

    public void setClientId(String clientId) {
        this.clientId = clientId;
    }

    public MqttClientPersistence getMqttClientPersistence() {
        return mqttClientPersistence;
    }

    public void setMqttClientPersistence(MqttClientPersistence mqttClientPersistence) {
        this.mqttClientPersistence = mqttClientPersistence;
    }
}

File: src/main/java/network/matic/dagger/Options.kt
package network.matic.dagger

import org.eclipse.paho.client.mqttv3.MqttClientPersistence
import org.eclipse.paho.client.mqttv3.MqttConnectOptions

open class Options {
    var clientId: String? = null
    var callback: Callback? = null
    var mqttConnectOptions: MqttConnectOptions? = null
    var mqttClientPersistence: MqttClientPersistence? = null
}